tests/test-cases/Group11-Upgrade/11-05-Configure.md

```diff
@@ -0,0 +1,16 @@
+Test 11-05 - Configure
+=======
+
+#Purpose:
```

Comment:
Minor: a space between `#` and the heading text renders the heading markdown correctly.